src/test/java/org/sagebionetworks/bridge/services/TemplateServiceTest.java

```diff
@@ -355,9 +398,16 @@ public void createTemplate() {
         assertEquals(template.getCreatedOn(), TIMESTAMP);
         assertEquals(template.getModifiedOn(), TIMESTAMP);
         assertEquals(template.getCriteria().getKey(), "template:"+GUID1);
+        assertEquals(template.getPublishedCreatedOn(), TIMESTAMP);
         
+        verify(mockTemplateRevisionDao).createTemplateRevision(revisionCaptor.capture());
         verify(mockCriteriaDao).createOrUpdateCriteria(criteria);
-        verify(mockTemplateDao).createTemplate(template);
+        verify(mockTemplateDao).createTemplate(eq(template), any());
+        
+        TemplateRevision revision = revisionCaptor.getValue();
+        assertEquals(revision.getSubject(), EMAIL_RESET_PASSWORD.name());
+        assertEquals(revision.getDocumentContent(), EMAIL_RESET_PASSWORD.name());
+        assertEquals(revision.getMimeType(), HTML);
```

Comment:
Should we verify the revision properties set by the create API?
* createdBy
* createdOn
* templateGuid
* storagePath

Reply:
Definitely, added.